src/bin/purge_db.rs

```diff
@@ -97,10 +97,6 @@ fn is_content_valid(content_key: &HistoryContentKey, value: &[u8]) -> bool {
         HistoryContentKey::BlockBody(_) => BlockBody::from_ssz_bytes(value).is_ok(),
         HistoryContentKey::BlockReceipts(_) => Receipts::from_ssz_bytes(value).is_ok(),
         HistoryContentKey::EpochAccumulator(_) => EpochAccumulator::from_ssz_bytes(value).is_ok(),
-        HistoryContentKey::Unknown(_) => {
```

Comment:
I think this change will break the script when on line 72 if an invalid content value is present.